docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -10,113 +10,155 @@ You can push {es-sec} cases to these third-party systems:
 * {ibm-r}
 * {swimlane}
 
+IMPORTANT: To create connectors and send cases to external systems, you need the
+https://www.elastic.co/subscriptions[appropriate license] and your role needs *All* privileges for the *Action and Connectors* feature. For more information, see <<case-permissions>>.
+
 To push cases, you need to create a connector, which stores the information
 required to interact with an external system.
 
 After you have created a connector, you can set {es-sec} cases to
 automatically close when they are sent to external systems.
 
-NOTE: To create connectors and send cases to external systems, you need the
-https://www.elastic.co/subscriptions[appropriate license] and your role needs *All* privileges for the *Action and Connectors* feature. For more information, see <<case-permissions>>.
-
-[[create-new-connector]]
 [float]
+[[create-new-connector]]
 === Create a new connector
 
 . Go to *Investigate* -> *Cases* -> *Edit external connection*.
 +
 [role="screenshot"]
 image::images/cases-ui-connector.png[Shows the page for creating connectors]
 . From the *Incident management system* list, select *Add new connector*.
-. Select one of these:
-* *{sn}*: To send cases to {sn}
+. Select the system to send cases to: *{sn}*, *{jira}*, *{ibm-r}*, or *{swimlane}*.
+
++
+IMPORTANT: If you've upgraded from {stack} version 7.15.0 or earlier to version 7.16.0 or later, you must install https://store.servicenow.com/sn_appstore_store.do#!/store/application/7148dbc91bf1f450ced060a7234bcb88[Elastic for ITSM] or https://store.servicenow.com/sn_appstore_store.do#!/store/application/2f0746801baeb01019ae54e4604bcb0f[Elastic for Security Operations (SecOps)] on your {sn} instance and complete additional configuration steps before creating a new {sn} ITSM or SecOps connector. For more information, refer to {kibana-ref}/servicenow-sir-action-type.html[{sn} SecOps] and {kibana-ref}/servicenow-action-type.html[{sn} ITSM].
+
+. Enter your required settings.
```

Comment:
This table contains _a lot_ of disparate info -- I'd imagine a user would only want information about the specific connector type they're working with, and could get frustrated having to weed out info about the other types.

Seems like a "[tabbed widget](https://github.com/elastic/docs#tabbed-widgets)" could be a good format for this, or at least just breaking into smaller, more focused tables for each connector type.